### DISCUSSION TITLE
Logo...?

### DISCUSSION THREAD

**Discussion by ioquatix** [axboe/liburing#700] [General]
Hi, I work with an artist making art work for open source projects. Would you like me to work on a logo for `liburing`? It won't cost the project anything because I'll put up the funds for it. The expectation is we'd defer to the artist for design, with a few revisions and then use it on the README and can be used in other places (normally released under one of the liberal CC licenses).

**Comment by axboe** [comment]
Definitely! I'd be interested in seeing some options, and agree on the licensing.

**Reply by ioquatix** [reply]
Okay, it won't be for a while but maybe at the end of this year.

**Reply by ioquatix** [reply]
FYI, here is a previous logo we did: https://github.com/rack/rack there are about 5 more in the pipeline.

**Reply by axboe** [reply]
That's fine, there's no rush. Always cool to have options.
